### PR TITLE
limel-checkbox for limel-form

### PR DESCRIPTION
**Pull request by jgroth**
Connected to Lundalogik/crm-feature#947 

### Browsers tested:

Windows:
- [ ] Chrome
- [ ] Firefox

Linux:
- [ ] Chrome
- [ ] Firefox

macOS:
- [ ] Chrome
- [ ] Firefox
- [ ] Safari

Mobile:
- [ ] Chrome on Android
- [ ] iOS

### CODE REVIEW

File: src/components/checkbox/checkbox.template.tsx [hunk 1]
```diff
@@ -36,7 +41,16 @@ export const CheckboxTemplate: FunctionalComponent<CheckboxTemplateProps> = prop
...
-            <label htmlFor={props.id}>{props.label}</label>
+            <label
+                class={{
+                    'mdc-checkbox--invalid': props.invalid,
+                    'mdc-checkbox--disabled': props.disabled,
+                    'mdc-checkbox--required': props.required,
+                }}
+                htmlFor={props.id}
+            >
+                {props.label}
```
Comment (adrianschmidt) [inline src/components/checkbox/checkbox.template.tsx]: Ew… I think prettier did an oopsie… 😣

File: src/components/checkbox/checkbox.template.tsx [hunk 2]
```diff
@@ -12,17 +14,20 @@ export const CheckboxTemplate: FunctionalComponent<CheckboxTemplateProps> = prop
...
+                    'mdc-checkbox--required': props.required,
+                }}
             >
                 <input
                     type="checkbox"
                     class="mdc-checkbox__native-control"
                     id={props.id}
                     checked={props.checked}
                     disabled={props.disabled}
+                    required={props.required}
```
Comment (adrianschmidt) [inline src/components/checkbox/checkbox.template.tsx]: Something makes the checkbox in the existing example required… 🤔

Comment (adrianschmidt) [reply src/components/checkbox/checkbox.template.tsx]: Ah… hehe… the checkbox in the example was explicitly set to required already 😂

Comment (adrianschmidt) [reply src/components/checkbox/checkbox.template.tsx]: Ah again… Now I see you set it in this PR. I missed that at first, for some reason. Sorry for spamming you with notifications 😄

File: src/examples/checkbox/checkbox.tsx [hunk 3]
```diff
@@ -27,6 +27,7 @@ export class CheckboxExample {
                         label="My fab checkbox"
                         id="fab"
                         checked={this.value}
+                        required={true}
```
Comment (adrianschmidt) [inline src/examples/checkbox/checkbox.tsx]: I'll make this switchable, just like `disabled`, if that's ok with you.

Comment (jgroth) [reply src/examples/checkbox/checkbox.tsx]: Yeah, of course! :smile:

File: src/components/checkbox/checkbox.template.tsx (hunk 1 above)
Comment (adrianschmidt) [reply src/components/checkbox/checkbox.template.tsx]: Scratch that… I'm just tired or something 🙄

**Pull request merged**